lib/tasks/accounts.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+namespace :accounts do
+
+  desc "Count bought cinstnaces for each account"
+  task count_bought_cinstances: :environment do
```

Comment:
Add a test in [/test/unit/tasks/...](https://github.com/3scale/porta/tree/master/test/unit/tasks)